From f84dbeffdf969f2b06d4bf6d4cbd6b49c77bb58d Mon Sep 17 00:00:00 2001
From: John Snow <jsnow@redhat.com>
Date: Wed, 20 Mar 2019 16:16:27 +0100
Subject: [PATCH 029/163] iotests: Remove superfluous rm from 232

RH-Author: John Snow <jsnow@redhat.com>
Message-id: <20190320161631.14841-16-jsnow@redhat.com>
Patchwork-id: 84957
O-Subject: [RHEL-7.7 qemu-kvm-rhev PATCH 15/19] iotests: Remove superfluous rm from 232
Bugzilla: 1668956
RH-Acked-by: Max Reitz <mreitz@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

From: Max Reitz <mreitz@redhat.com>

This test creates no such file.

Signed-off-by: Max Reitz <mreitz@redhat.com>
Reviewed-by: Eric Blake <eblake@redhat.com>
Reviewed-by: John Snow <jsnow@redhat.com>
Message-id: 20190210145736.1486-4-mreitz@redhat.com
Signed-off-by: Max Reitz <mreitz@redhat.com>
(cherry picked from commit 8f4ed6983ab1bda264074ac98d32657b411223bc)
Signed-off-by: John Snow <jsnow@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 tests/qemu-iotests/232 | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/qemu-iotests/232 b/tests/qemu-iotests/232
index bc2972d..de884cd 100755
--- a/tests/qemu-iotests/232
+++ b/tests/qemu-iotests/232
@@ -30,7 +30,6 @@ status=1	# failure is the default!
 _cleanup()
 {
     _cleanup_test_img
-    rm -f $TEST_IMG.snap
 }
 trap "_cleanup; exit \$status" 0 1 2 3 15
 
-- 
1.8.3.1

